### DISCUSSION TITLE
[Feature] shared albums: number of contributors, show the contributor of assets

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6197] [Feature Request]
### The feature

I would like to see, if albums contains content from multiple contributors. The best would be to show the number of contributors.
For example, currently the information is "x items . owned", this could be extended by the number of contributors, if this number is > 1.

If an album has multiple contributors, or if it is shared by someone else, then inside the album, I want to see the contributor of each asset. Google photos do this in a pragmatic way: if there are multiple contributors, then each photo indicates the contributor: in thumbnails and also in the one-picture-view (in details).

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by aisbergde** [comment]
v1.92.0 web client is showing "shared by" in the details. This is fine.
The suggested next step would be to show the "shared by" information also in the thumbnail view of the album (or in general in the timeline view, when it will be implemented to show shared assets in the timeline)

And I would suggest to show information "shared by" also in the mobile client.

**Comment by bo0tzz** [comment]
#8779
